lib/ofx/parser/datetime.ex

```diff
@@ -1,67 +1,127 @@
 defmodule Ofx.Parser.Datetime do
   @moduledoc false
 
-  import String, only: [to_integer: 1]
-
   alias Ofx.Parser.Error
 
-  @regex ~r'^(?<y>\d{4})(?<M>\d{2})(?<d>\d{2})(?<h>\d{2})?(?<m>\d{2})?(?<s>\d{2})?(\.\d{3})?.*?(\[(?<signal>[-+]?)(?<th>\d{1,2})\.?(?<tm>\d{1,2})?)?'
-
-  @seconds_in_an_hour 3600
-  @seconds_in_a_minute 60
+  defstruct [:input, :datetime, :date, :time, :offset, :timezone, :zone_abbreviation]
```

Comment:
```suggestion
  defstruct [:input, :datetime_string, :date, :time, :offset, :timezone, :zone_abbreviation]
```
Only at the end the string is converted to a datetime. Should we signalized that in the name?